ansible/templates/common/gunicorn.j2

```diff
@@ -13,4 +13,14 @@ Additional settings can be configured by changing the following properties:
 - :property:`omero.web.wsgi_args` Additional arguments. For more details
   check `Gunicorn Documentation <https://docs.gunicorn.org/en/stable/settings.html>`_.
 
-{% endif %}
\ No newline at end of file
+{% else %}
+#!/bin/bash
+
+set -e -u -x
+export PATH=/opt/omero/web/venv3/bin:$PATH
+
+omero config set omero.web.application_server.max_requests 500
```

Comment:
Is this needed?

Reply:
This is only to apply what we describe in the doc

Reply:
There's no sign of it on any of our prod-playbook servers

Reply:
In that case we could remove that from the doc